x-pack/plugins/ml/public/components/job_select_list/job_select_button.html

```diff
@@ -1,14 +1,17 @@
 <div style='display: flex;' class="ml-job-select-btn-container">
-  <div class="ml-job-select-btn-label">Job</div>
+  <div class="ml-job-select-btn-label"
+    i18n-id="xpack.ml.jobSelectList.menuButtonLabel"
+    i18n-default-message="Job"
+  ></div>
   <div style="flex:0 0 auto;">
     <div  popover-placement="bottom"
           popover-placement="bottom"
           popover-html-unsafe="{{unsafeHtml}}"
           popover-append-to-body="false"
           popover-trigger="click"
           class="ml-job-select-btn"
-          tooltip="{{description.txt}} selected"
-          aria-label="Job selection menu"
+          tooltip="{{ ::'xpack.ml.jobSelectList.menuButtonTooltip' | i18n: { defaultMessage: '{descriptionText} selected', values: {descriptionText: description.txt} } }}"
```

Comment:
For some reason `descriptionText: description.txt` is not binding to the UI (returns empty string).

Reply:
I believe that is because you use one time binding operator `::` when you pass some variable to values then you must not use one time binding